covfee/server/orm/annotator.py

```diff
@@ -0,0 +1,43 @@
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+import datetime
+from .base import Base
+from typing import Optional
+
+
+class Annotator(Base):
+    """
+    Links an annotator to a single journey instance
+    """
+
+    __tablename__ = "annotators"
+
+    # A unique identifier for the annotator row
+    id: Mapped[int] = mapped_column(primary_key=True)
```

Comment:
Do we need the `id` if we already have the `prolific_id`?

Reply:
not completely. However, I'm thinking of future proofing as an annotator could have a prolific_id or an mturk id .. or something else. Thus intentionally keeping the Annotator concept as a super set of prolific academic and others.